core/templates/core/reported-post.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'core/css/style.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="container mt-5">
+        <h1 class="mb-4">Reported Post</h1>
```

Comment:
Here you use English, in the post-report.html it was Polish... Please check the whole PR and use English everywhere